tasks/main.yml

```diff
@@ -6,7 +6,17 @@
 
 - name: Install Hipache packages.
   apt: update_cache=true name="{{ item }}"
-  with_items: packages
+  with_items:
+    - python-software-properties
+    - nodejs
+
+- name: Install latest version of node-hipache
+  apt: name=node-hipache
+  when: node_hipache_version is not defined
```

Comment:
Can this be a single task like https://github.com/alphagov/ansible-playbook-docker_server/commit/3a87af4cde8bd8c9a53c8b0e5aaf1d5245eb789a#diff-2444ad0870f91f17ca6c2a5e96b26823R24 for code and output simplicity?